pkg/virt-handler/virtwrap/api/converter.go

```diff
@@ -287,9 +333,32 @@ func Convert_v1_VirtualMachine_To_api_Domain(vm *v1.VirtualMachine, domain *Doma
 		volumes[volume.Name] = volume.DeepCopy()
 	}
 
-	for _, disk := range vm.Spec.Domain.Devices.Disks {
+	machine := "pc" // QEMU default
+	if vm.Spec.Domain.Machine != nil {
```

Comment:
If `Spec.Domain.Machine` is `nil` would it make sense to initialize it?

Reply:
we handle Spec.Domain.Machine properly in line 402. I added this (minor) duplication to keep this patch minimal, I'd like some refactoring later (next upload or future PR as reviewers prefer)